orchestra/models/communication/mixins.py

```diff
@@ -66,3 +67,35 @@ def __str__(self):
             self.is_available,
             self.is_winner
         )
+
+    def _mark_staffbot_request_complete(self, request):
+        from orchestra.models import StaffBotRequest
+        request.status = StaffBotRequest.Status.COMPLETE.value
+        request.save()
+
+    def save(self, *args, **kwargs):
```

Comment:
not a strong opinion: would this this logic more cleanly live in whatever functions get called when a worker responds accept/ignore rather than in this save method?